backend/app/alembic/versions/dc391733e10a_add_extent_to_gerrydb.py

```diff
@@ -0,0 +1,117 @@
+"""add extent to gerrydb
+
+Revision ID: dc391733e10a
+Revises: 5ab466c5650a
+Create Date: 2024-10-27 12:22:30.468315
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from app.models import GerryDBTable
+
+# revision identifiers, used by Alembic.
+revision: str = "dc391733e10a"
+down_revision: Union[str, None] = "5ab466c5650a"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    # Make column nullable to avoid errors during initial insert
+    op.add_column(
+        "gerrydbtable", sa.Column("extent", sa.ARRAY(sa.Float()), nullable=True)
+    )
+
+    # Use a session to interact with the database
+    bind = op.get_bind()
```

Comment:
I: I think it's a good idea to create these bboxes but was wondering if we should [align all geometry columns to "geometry"](https://github.com/districtr/districtr-v2/pull/148) to simplify this logic.